gh_pages/_source/session5/Building-a-Perception-Pipeline.rst

```diff
@@ -32,60 +27,68 @@ We will create a new catkin workspace, since this exercise does not overlap with
 
     .. code-block:: bash
 
-			catkin init
-			catkin build
+			colcon build
 
  #. Source the workspace
 
     .. code-block:: bash
 
-      source ~/perception_ws/devel/setup.bash
+      source ~/perception_ws/install/setup.bash
```

Comment:
Should this be done in a different terminal to avoid doubling up paths and such?